src/os/bluestore/BitAllocator.h

```diff
@@ -19,6 +18,18 @@
 #include "include/intarith.h"
 #include "os/bluestore/bluestore_types.h"
 
+#define alloc_assert assert
+
+#ifdef BIT_ALLOCATOR_DEBUG
+#define alloc_dbg_assert(x) assert(x)
+#else
+#define alloc_dbg_assert(x) noop(x)
+#endif
+
+static inline void noop(bool var)
+{
+}
```

Comment:
@liewegas , changed dummy function to what you suggested from assert.h. Please have a look now. I also wanted to define this macro BIT_ALLOCATOR_DEBUG for test case compilation but could not make it work :(.